shared/simulator/src/vm/mod.rs

```diff
@@ -227,8 +228,10 @@ impl TeamController {
             slice.write_slice(&state.state).expect("system state write");
         }
 
+        // Run user's ship tick
         let result = vm.tick_ship.call(vm.store_mut().deref_mut(), &[]);
```

Comment:
I think this line calls into `shared/ai/src/tick.rs`. Is that right?

Reply:
Right.